tests/test_lowlevel.py

```diff
@@ -1105,7 +1105,7 @@ def test_bad_parameters(self):
         with pytest.raises(_msprime.InputError):
```

Comment:
Line 1104 here is still "Track"

Reply:
Thanks! I missed it due to the capital letter "T".